extension/src/popup/views/FullscreenSuccessMessage.tsx

```diff
@@ -50,27 +60,78 @@ const IlloContainerEl = styled.div`
   padding: 1rem 0;
 `;
 
+const MessageEl = styled.div`
+  border-radius: ${ROUNDED_CORNERS};
+  background-color: #563bf1;
+  padding: 1.75rem 1.5rem;
+  text-align: left;
+  font-size: 0.93rem;
+  margin: 1.25rem 0 0;
+`;
+
+const IconEl = styled.img`
+  display: inline-block;
+  margin-top: -0.4rem;
+  max-width: 1.875rem;
+  max-height: 1.875rem;
+  vertical-align: middle;
+`;
+
+const MessageSubheaderEl = styled.h2`
+  color: ${COLOR_PALETTE.white};
+  display: inline-block;
+  font-weight: ${FONT_WEIGHT.bold};
+  font-size: 1.1rem;
+  padding-left: 0.65rem;
+  margin: 0;
+`;
+
+const MessageListEl = styled.ul`
+  padding: 0 1.3rem;
+
+  li {
+    margin-top: 1rem;
+  }
+`;
```

Comment:
this alert message layout is very *similar* to what we do in `<WarningMessage />`, but just different enough that it doesn't really make sense to share any code. Kinda annoying.

This whole app would benefit from a design based on a system. Something I'll push for with the design team for v2